lib/threema/send/file.rb

```diff
@@ -20,6 +20,7 @@ def structure(params)
         return content if !params[:thumbnail]
```

Comment:
maybe we could pass through the rendering type options to the users of the gem here instead of inferring it from a set thumbnail